### PR TITLE
feat: [Issuer Adapter] Sign Consent Credential

### LINKED ISSUE #180: [Issuer Adapter] Sign Consent Credential

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #180 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: pkg/restapi/issuer/operation/operations.go [hunk 1]
```diff
@@ -209,12 +209,25 @@ func (o *Operation) createIssuerProfileHandler(rw http.ResponseWriter, req *http
 		return
 	}
 
+	if len(newDidDoc.AssertionMethod) == 0 {
```
Comment (llorllale) [inline pkg/restapi/issuer/operation/operations.go:212]: have you tested this in a didcomm flow?

Comment (llorllale) [reply pkg/restapi/issuer/operation/operations.go:212]: We are using the peer DIDs exchanged during did-exchange.

File: pkg/restapi/rp/operation/testdata/context/consent-credential-v1.jsonld [hunk 2]
```diff
@@ -5,45 +5,32 @@
       "name":"http://schema.org/name",
```
Comment (llorllale) [inline pkg/restapi/rp/operation/testdata/context/consent-credential-v1.jsonld:5]: there's another copy of this file in the BDD tests folder

Comment (rolsonquadras) [reply pkg/restapi/rp/operation/testdata/context/consent-credential-v1.jsonld:5]: @llorllale [done](https://github.com/trustbloc/edge-adapter/compare/e6c1f198f689cad99da478f8e1ef9edd349f4c8e..8eeb9c4be51ebc0319672eca4cb4ae432d1bfb3b).

**Pull request merged**